rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.78"
+components = ["rustfmt", "rust-src", "clippy"]
+targets = ["wasm32-unknown-unknown"]
```

Comment:
Would this also install the target specific the machine the compiler is running on?

Reply:
Yes, the host target is always implied, the target list here are "additional targets" above and beyond that default target.